patterns/test/test_incident_analysis/ClusterAnalysisTestCase.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
```

Comment:
test_incident_analysis stuff looks like a stub/work in progress so stopped reviewing from here down